clang/test/compiler-course/kudryashova_i_test_clangast/kudryashova_i_clangast_test.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/Implicit_Conv_Kudryashova_Irina_FIIT3_ClangAST%pluginext -plugin ImplicitConvPlugin %s -fsyntax-only 2>&1 | FileCheck %s
+
+// CHECK: Function `sum`
+// CHECK-NEXT: float -> double: 1
+// CHECK-NEXT: int -> float: 1
```

Comment:
It is better to insert checks immediately before the tested code
```suggestion
// CHECK: Function `sum`
// CHECK-NEXT: float -> double: 1
// CHECK-NEXT: int -> float: 1
double sum(int a, float b) {
	return a + b;
}
```